src/report_generator.cpp

```diff
@@ -388,6 +408,12 @@ R"(<table>
    out << "<p>" << build_timestamp << "</p>";
 }
 
+std::string prune_title_tags(const std::string& title){
```

Comment:
No, I'm pretty sure that we shouldn't depend on such old gcc versions, the <regex> dependency would be fine!